cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -98,8 +98,14 @@ ONEDAL_EXPORT result_option_id get_responses_id();
 /// what should algorithm return
 namespace result_options {
 
+/// Signs that algorithm must return nearest neighbors indices
 const inline result_option_id indices = detail::get_indices_id();
+
+/// Signs that algorithm must return nearest neighbors distances
 const inline result_option_id distances = detail::get_distances_id();
+
+/// Signs that algorithm must return results of :expr<classification>
```

Comment:
`Signs that algorithm must` is incorrect, I suggest this:

```suggestion
/// Return the indices of the nearest neighbors
const inline result_option_id indices = detail::get_indices_id();

/// Return the distances to the nearest neighbors
const inline result_option_id distances = detail::get_distances_id();

/// Return the :expr<classification> results
```